x/fees/genesis.go

```diff
@@ -22,8 +22,10 @@ func InitGenesis(
 		withdrawal := sdk.MustAccAddressFromBech32(fee.WithdrawAddress)
```

Comment:
@fedekunze @ramacarlucho Should we rename withdraw to withdrawer? Might sound better on the endpoints, e.g. WithdrawerFees

Reply:
sure 👍